exo-go/src/cmd/add_template.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/Originate/exosphere/exo-go/src/os_helpers"
+	"github.com/Originate/exosphere/exo-go/src/template_helpers"
+	"github.com/Originate/exosphere/exo-go/src/user_input_helpers"
+	"github.com/Originate/exosphere/exo-go/src/util"
+	"github.com/spf13/cobra"
+)
+
+var addTemplateCmd = &cobra.Command{
+	Use:   "add-template",
+	Short: "Adds a remote service template to .exosphere",
+	Long:  "Adds a remote service template to .exosphere",
+	Run: func(cmd *cobra.Command, args []string) {
+		if util.PrintHelpIfNecessary(cmd, args) {
+			return
+		}
+		if len(args) != 2 {
+			fmt.Println("not enough arguments")
+			os.Exit(1)
+		}
+		fmt.Print("We are about to add a new service template\n")
+		templateName, gitURL := args[0], args[1]
+		templateDir := path.Join(".exosphere", templateName)
+		if osHelpers.DirectoryExists(templateDir) {
+			reader := bufio.NewReader(os.Stdin)
+			query := fmt.Sprintf(`The template "%s" already exists, do you want to update its git URL to %s`, templateName, gitURL)
```

Comment:
Kevin just told me to get rid of this overwriting feature (i.e. just abort when users try to add an existing service). So this is no longer relevant.